pairinteraction_gui/pairinteraction/app.py

```diff
@@ -483,21 +483,34 @@ def __init__(self, parent=None):
         self.path_cpp_complex = os.path.join(self.path_base, self.path_workingdir, "pairinteraction-complex")
         self.path_quantumdefects = os.path.join(self.path_base, self.path_workingdir, "databases/quantum_defects.db")
 
-        if os.name == "nt":
-            self.path_out = os.path.join(self.userpath, "pairinteraction/")
+        if sys.platform == "win32":
+            self.path_config = os.path.expandvars(r"%APPDATA%\pairinteraction")
+            if r"%APPDATA%" in self.path_config:
+                self.path_config = os.path.expanduser(r"~\AppData\Local\pairinteraction")
```

Comment:
Shouldn't it be `self.path_config = os.path.expanduser(r"~\AppData\Roaming\pairinteraction")`?